conf/declaration.routes

```diff
@@ -533,7 +533,9 @@ POST        /containers/:containerId/seals/:sealId/remove    controllers.declara
 
 # Summary
 
-GET         /saved-summary                       controllers.declaration.SummaryController.displayPage
+GET         /summary                             controllers.declaration.SummaryController.displayPage
```

Comment:
Do we need to rename this endpoint? Just any tickets that reference this url will become stale so wondering if this is really needed? :thinking:

Reply:
Just to be more consistent with other related endpoints. In addition `saved-summary` was also sort of misleading IMO